src/SplitIO/Sdk/Client.php

```diff
@@ -68,6 +75,15 @@ private function logImpression(
             $changeNumber,
             $bucketingKey
         );
+
+        return $impression;
+    }
+
+    /**
+     * @param $impression \SplitIO\Sdk\Impressions\Impression
+     */
+    private function logImpression(Impression $impression)
```

Comment:
do we need this function?

Reply:
No, we don't. I will remove it and call `Treatment.log()`